### PR TITLE
Add Tests

### PR DESCRIPTION
**Pull request by GuillaumeRx**
[Story details](https://app.clubhouse.io/mycrypto/story/6339/)

- [x] Setup TestCafe

- [x] Add Homepage rendering tests

### CODE REVIEW

File: __tests__/jest.config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = {
     '^.+\\.[jt]sx?$': `<rootDir>/__tests__/jest-preprocess.js`,
   },
   testRegex: '(/__tests__/.*|(\\.|/)(test|spec))\\.(jsx?|tsx?)$',
-  moduleDirectories: ['node_modules', 'src'],
+  moduleDirectories: ['node_modules', 'src', '<rootDir>/__tests__'],
```
Comment (vilsbole) [inline __tests__/jest.config.js:11]: I doubt jest should be reaching into the E2E folder.
Is this necessary?

File: .testcaferc.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "browsers": ["chrome"],
+  "src": ["./__tests__/*.test.js"],
+  "proxyByPass": ["localhost:3000"],
+  "clientScripts": [
+    { "module": "@testing-library/dom/dist/@testing-library/dom.umd.js" }
+  ],
+  "developmentMode": true
```
Comment (vilsbole) [inline .testcaferc.json:8]: Is this necessary?

File: .testcaferc.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+  "browsers": ["chrome"],
+  "src": ["./__tests__/*.test.js"],
+  "proxyByPass": ["localhost:3000"],
```
Comment (vilsbole) [inline .testcaferc.json:4]: Not the same port I believe

File: .testcaferc.json (hunk 2 above)
Comment (GuillaumeRx) [reply .testcaferc.json:8]: per : https://devexpress.github.io/testcafe/documentation/reference/configuration-file.html#developmentmode

No needs, it's only for debugging with TestCafe team

Comment (GuillaumeRx) [reply .testcaferc.json:8]: We should also remove it from https://github.com/MyCryptoHQ/MyCrypto

File: .github/workflows/mycryptobuilds.yml [hunk 4]
```diff
@@ -128,13 +135,25 @@ jobs:
...
 
     steps:
       - name: Checkout PR
         uses: actions/checkout@v1
 
+      # Define url as env var to be used by TestCafe.
+      - name: Set E2E_BASE_URL
+        if: github.ref == 'refs/heads/master'
+        run: |
+          E2E_BASE_URL=$( echo https://landing.mycryptobuilds.com/master/# )
```
Comment (vilsbole) [inline .github/workflows/mycryptobuilds.yml:153]: hash leads to not found

**Pull request merged**